diff -Naur xf86-input-keyboard-1.6.2/src/Makefile.am xf86-input-keyboard-1.6.2.tpg/src/Makefile.am
--- xf86-input-keyboard-1.6.2/src/Makefile.am	2013-03-08 16:30:28.149164498 +0000
+++ xf86-input-keyboard-1.6.2.tpg/src/Makefile.am	2013-03-08 16:30:07.000000000 +0000
@@ -23,7 +23,7 @@
 kbd_drv_la_LTLIBRARIES = kbd_drv.la
 kbd_drv_la_LDFLAGS = -avoid-version -module
 kbd_drv_la_SOURCES = kbd.c xf86OSKbd.h xf86Keymap.h atKeynames.h
-kbd_drv_la_LIBADD = $(XORG_LIBS)
+kbd_drv_la_LIBADD = $(XORG_LIBS) -lxi
 kbd_drv_ladir = @inputdir@
 
 BSD_SRCS = bsd_KbdMap.c bsd_kbd.c bsd_kbd.h at_scancode.c
